chain/network/Cargo.toml

```diff
@@ -17,6 +17,7 @@ rand = "0.6.5"
 reed-solomon-erasure = "3.1.1"
 byteorder = "1.2"
 lazy_static = "1.4"
+itertools = "0.8.0"
```

Comment:
We intentionally avoid using this to reduce dependencies. Unless there is a strong reason we should not use it.

Reply:
Ohh, sad to hear that, this allowed me to make some idiomatic piece of code, which is cleaner with itertools than without it, but of course it is not mandatory. I can do it by myself.